aws/readme.md

```diff
@@ -7,16 +7,66 @@ After completing these steps, please follow the required steps in the [project r
 1. Create AWS secret keys and provide them to terraform https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables
 2. `terraform init`
 3. `terraform apply` to create the resources on AWS.
-    * Supported configuration variables (See `vars.tf`):
-        * `aws_region`: The AWS region to deploy the infrastructure on. Default: `us-west-2`.
-        * `nodes`: The number of nodes to base the cluster on. Default: `1`.
-        * `enable_monitoring`: Will create a prometheus/grafana instance to be used for monitoring the cluster. Default: `true`.
-        * `instance_type`: The instance type which redpanda will be deployed on. Default: `i3.8xlarge`.
-        * `prometheus_instance_type`: The instance type which prometheus and grafana will deployed on. Default: `c5.2xlarge`.
-        * `public_key_path`: Provide the path to the public key of the keypair used to access the nodes. Default: `~/.ssh/id_rsa.pub`
-        * `distro`: Linux distribution to install (this settings affects the below variables). Default: `ubuntu-focal`
-        * `distro_ami`: AWS AMI to use for each available distribution.
-        These have to be changed with according to the chosen AWS region.
-        * `distro_ssh_user`: User used to ssh into the created EC2 instances.
-
-  Example: `terraform apply -var="instance_type=i3.large" -var="nodes=3"`
+  
+Example: `terraform apply -var="instance_type=i3.large" -var="nodes=3"`
+
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| aws | 3.29.1 |
+| local | 2.1.0 |
+| random | 3.1.0 |
+
+## Providers
```

Comment:
Should we merge requirements and providers?

Reply:
Defining the versions supported  we ensure that the code will work with those versions

Reply:
Yeah I agree with that but is there a reason to track it both under requirements and providers?

Reply:
Maybe not, currently, the code is very clean and just deployy some instances, in the future if the code depends on a specific feature, we can contemplate merging providers and requirements. I could remove providers and requirements, what do you think?

Reply:
I'm fine with keeping on or the other since they point to exactly the same versions :+1: .